CHANGELOG.md

```diff
@@ -1851,7 +1871,6 @@ Other minor changes
   ∈-++-comm : ∀ {v} xs ys → v ∈ xs ++ ys → v ∈ ys ++ xs
   ∈-++-comm∘++-comm : ∀ {v} xs {ys} (p : v ∈ xs ++ ys) → ∈-++-comm ys xs (∈-++-comm xs ys p) ≡ p
   ∈-++↔++ : ∀ {v} xs ys → v ∈ xs ++ ys ↔ v ∈ ys ++ xs
-  ```
```

Comment:
Is there a reason you've deleted this line?

Reply:
Nope; restored in next push.